24.01.03_튜플ㅋ.js

```diff
@@ -0,0 +1,27 @@
+function solution(s) {
+    let obj1 = {},
+        arr1 = [],
+        arr2 = [],
+        anwser,
+        anwser1;
+    let arr = s.slice(2, s.length - 2).split('},{');
+    let map1 = arr.map((data) => {
+        const temp = data.split(',');
+        const temp2 = temp.map((x) => Number(x));
+        return temp2;
```

Comment:
```suggestion
        const temp = data.split(',').map((x) => Number(x));
        return temp
```
나눠서 하기 이전에 이런식으로 하나의 로직으로 구섷아면 한 줄로 간결하게 표현이 가능합니다.
이제 이 로직이 어떤 방식으로 돌아가는지 이해가 되겠죠?